representatives/contrib/francedata/variants.py

```diff
@@ -110,6 +151,31 @@ def _get_an_comittees(json):
     return comittees
 
 
+def _get_an_delegations(json):
```

Comment:
Perhaps it would be nice to rename the argument to a name that's not a standard library to avoid potential import shadowing ?